cmd/peer-rest-common.go

```diff
@@ -16,7 +16,7 @@
 
 package cmd
 
-const peerRESTVersion = "v5"
+const peerRESTVersion = "v6"
```

Comment:
Can you mention about this version change in the PR and commit as well.

Reply:
:+1:  done